### PR TITLE
 #164200170 Fix bug in login feature and adding toast message

### PR DESCRIPTION
**Pull request by Georgeygigz**
**What does this PR do?**

* It fixes a bug where it allows users to enter the correct credentials without refreshing the page.
* added a toaster message

**How should this be manually tested?**
* clone this repo `https://github.com/andela/ah-django-unchained-frontend`
* `cd ah-django-unchained-frontend`
* `git checkout bg-fix-login-page-bug-164200170`
* Install the required dependencies `npm install`
* `npm start to run the local server`

**Pivotal Tracker story**
[#162948804](https://www.pivotaltracker.com/story/show/164200170)

**Screenshot**
image

image

### CODE REVIEW

File: src/pages/LoginPage/index.jsx [hunk 1]
```diff
@@ -28,70 +29,72 @@ export class LoginPage extends Component {
...
+                      <Button
+                        type="submit"
+                        className="btn btn-info btn-block"
+                        text="Log In"
+                      />
+                      <p className="d-flex justify-content-center">
+                        Dont have an account? Sign up here
+                      </p>
+                      <p className="d-flex justify-content-center">
+                        <a href=" ">
```
Comment (kwameasiago) [inline src/pages/LoginPage/index.jsx:84]: Why do you have an empty a tag

**Review by MaggieChege — APPROVED** [review]
Looks good to me.

File: src/pages/LoginPage/index.jsx [hunk 2]
```diff
@@ -28,70 +29,72 @@ export class LoginPage extends Component {
...
+                          <Link to="/passwordreset">
+                            Click here Forgot password? Reset it here.
```
Comment (andrewhingah) [inline src/pages/LoginPage/index.jsx]: Kindly change this text to `Forgot password? Click here`

**Comment by andrewhingah** [comment]
Good job @Georgeygigz here. But kindly make the title of this PR more descriptive. Mention the bug you are fixing.

**Review by cosmas28 — CHANGES_REQUESTED** [review]
Kindly implement the toast to the `signup` and `socialAuth` features.

**Comment by Georgeygigz** [comment]
> Kindly implement the toast to the `signup` and `socialAuth` features.

I was helping @maggy. I am working on this right away.

File: src/pages/LoginPage/index.jsx (hunk 1 above)
Comment (Georgeygigz) [reply src/pages/LoginPage/index.jsx:84]: I was not working on this, My work was to add toast message only.

**Review by cosmas28 — APPROVED** [review]

**Pull request merged**